src/editor/editor.cpp

```diff
@@ -425,4 +425,18 @@ Editor::delete_markers() {
   inputcenter.delete_markers();
 }
 
+void
+Editor::change_tileset() {
+  tileset = TileManager::current()->get_tileset(level->get_tileset());
+  tileselect.input_type = EditorInputGui::IP_NONE;
+  for(auto& sector : level->sectors) {
```

Comment:
I wonder whether this iterator and the iterator below that also work with const auto&